conf/cassandra.yaml

```diff
@@ -1599,9 +1599,6 @@ drop_compact_storage_enabled: false
 #         warn_threshold_kb: 0
 #         abort_threshold_kb: 0
 
-# Whether guardrails are enabled or not. Guardrails are disabled by default.
```

Comment:
This got pulled in by mistake during rebase this morning; can ignore as I'll squash it out before merge.